01 - Navigation/01 - Navigate to something you know/Code/JetBrains/Customers/Customer.php

```diff
@@ -20,7 +20,7 @@ function __construct($name, $age)
     }
 
     /**
-     * @param int $age
+     * @param $age int
```

Comment:
Perhaps it's best to wait for @maartenba to respond.

If you intend to add the missing newlines to all files, I think it's easier to do so with some simple scripting, like `sed` (for an example, see: http://unix.stackexchange.com/questions/31947/how-to-add-a-newline-to-the-end-of-a-file).